README.md

```diff
@@ -49,7 +49,7 @@ As an example, you might configure the library like this:
 ```js
 const momo = require("mtn-momo");
 
-const { Collections } = momo({ callbackHost: process.env.CALLBACK_HOST });
+const { Collections, Disbursements } = momo({ callbackHost: process.env.CALLBACK_HOST });
 ```
 
 Currently, the library only supports `Collections`. Let us create a client with the previously generated credentials for the `Collections` product.
```

Comment:
We need to update this to reflect that the library supports both `Collections` and `Disbursements`